nodejs/overview/msteams-client.md

```diff
@@ -1,61 +1,111 @@
 ---
-title: Microsoft Teams JavaScript client library
+title: Microsoft Teams Client API reference
 description: Reference documentation for the latest Microsoft Teams JavaScript client library
-keywords: msteams teams client sdk javascript library reference latest
+ms.date: 01/03/2023
+ms.topic: reference
+keywords: msteams live-share teamsfx teams client sdk javascript library reference latest
 ---
-# Microsoft Teams JavaScript client library
+# Microsoft Teams Client API reference
 
-Starting with *@microsoft/teams-js* version `2.0.0`, the Microsoft Teams JavaScript client library has been refactored to enable [Teams apps to run in Outlook and Office](/microsoftteams/platform/m365-apps/overview), in addition to Microsoft Teams. From a functional perspective, the TeamsJS SDK is a superset of TeamsJS 1.x versions: it supports existing Teams app functionality while adding the ability to host Teams apps in Outlook and Office.
+This page contains general information about the client library to support you in the development of using The Microsoft Teams Client SDK.
 
-> [!TIP]
-> Refer to the [Teams JavaScript client library](/microsoftteams/platform/tabs/how-to/using-teams-client-library) overview for guidance on building both Teams-only and Microsoft 365-enabled apps and for current SDK capability support across Teams, Outlook, and Office.
+| Library | Description | APIs |
+|-|-|-|
+| **Live Share** | Live Share transforms the Microsoft 365 platforms to be more a collaborative experience without writing any dedicated back-end code | [Live Share](../docs-ref-autogen/%40microsoft/live-share/index.yml)<br/><br/>[Live Share canvas](../docs-ref-autogen/%40microsoft/live-share-canvas/index.yml)<br/><br/>[Live Share media](../docs-ref-autogen/%40microsoft/live-share-media/index.yml) |
+| **TeamsJS** | Teams Java script client library enables developers to create app content hosted in [IFrame](https://developer.mozilla.org/docs/Web/HTML/Element/iframe) across the Microsoft 365 ecosystem | [TeamsJS ](../docs-ref-autogen/%40microsoft/teams-js/index.yml) |
+| **TeamsFx** | TeamsFx helps to reduce tasks by using Microsoft Teams single sign-on (SSO) and accessing cloud resources down to single line statements with little configuration | [Teamsfx](../docs-ref-autogen/%40microsoft/teamsfx/index.yml) |
 
-## Finding the library
+## Microsoft Teams Live Share
 
-The TeamsJS client is distributed as an npm package. The latest version can be found here:
+The [Live share SDK]( https://github.com/microsoft/live-share-sdk) is enabled to allow for collaborative moments across Microsoft Teams through `sidePanel` and `meetingStage`, media synchronization for HTML `<video>` and `<audio>` elements, and annotation tools for whiteboard and PowerPoint presentations. 
```

Comment:
It would be great to explain here the relationship between the three LiveShare libraries here--which one(s) you would use for different scenarios. Also is LiveShare a superset of LiveShare Canvas + LiveShare Media?

Reply:
"...is enabled to allow for collaborative moments across Microsoft teams..." is a weird sentence to me. I can see a library enabling scenarios, but not the library itself being enabled if that makes sense. "...enables developers to create collaborative moments..." maybe?